blog/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.text import slugify
+
+from core.models import BaseModel
+
+
+class Post(BaseModel):
+    title = models.CharField(max_length=200)
+    slug = models.SlugField(
+        blank=True,
+        max_length=250,
+        null=True,
+        unique_for_date="published_date",
+    )
+    content = models.TextField()
+    author = models.ForeignKey(User, blank=True, on_delete=models.CASCADE, null=True)
```

Comment:
Do we want to delete the blog post if the author is deleted?

Reply:
Are users ever hard deleted, or are they deactivated? If they aren't ever deleted then I'm not sure how much this would matter, but if they are deleted then there would be a risk that articles would be deleted when the user was deleted (but I'm pretty sure the admin would give a warning about the associated records, right?)

Reply:
I changed this to an optional char field. I don't have it exposed in the templates, but this at least addresses one concern.